inference-engine/src/mkldnn_plugin/utils/blob_dump.h

```diff
@@ -19,15 +19,22 @@ namespace MKLDNNPlugin {
  * NB! Channel is a second dimension for all blob types.
  */
 class BlobDumper {
-    InferenceEngine::Blob::Ptr _blob;
-    InferenceEngine::Blob::Ptr _scales;
+    MKLDNNMemoryPtr memory;
+    // need to store data after read
+    std::vector<uint8_t> data;
```

Comment:
We do not need the internal buffer, just create memory with nullptr and all the internal memory allocation will be done by the memory object itself.